engine/src/fieldhtml.cpp

```diff
@@ -545,6 +551,10 @@ static void export_html_begin_lists(export_html_t& ctxt, uint32_t p_new_style, u
 		else
 			/* UNCHECKED */ MCStringAppendFormat(ctxt.m_text, "<li value=\"%d\">", p_index);
 	}
+	
+	// listStyle = "skip" does need a <li> tag as well
+	if (p_new_style == kMCParagraphListStyleSkip)
```

Comment:
@livecodepanos is there a reason not to just drop the `p_new_style != kMCParagraphListStyleSkip` above?

Reply:
@montegoulding just dropping `p_new_style != kMCParagraphListStyleSkip ` causes data loss in this use-case:
```
put "one" & cr & "two" & cr & "three" into field 1
set the listStyle of line 1 of field 1 to "skip"
set the listStyle of line 2 of field 1 to "disc"
set the htmltext of field 1 to the htmltext
```
as in the first pass `ctxt . list_depth` equals `p_new_depth` thus the code in the `if` block will not be executed, but we do need a `<li>` tag in this case anyway I think

Reply:
hmm... isn't that a general style of list changing but depth remaining the same issue? So style changes we should end the previous list and start a new list?